### ISSUE TITLE
help menu link (Trac #34)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#38]
-make a link in the help menu that opens a browser pointed to the wiki
so that people have easy access to documentation

Migrated from http://microfluidics.utoronto.ca/ticket/34
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "-make a link in the help menu that opens a browser pointed to the wiki\nso that people have easy access to documentation\n", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "help menu link", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-01-06T21:39:42", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "enhancement"
}
```

**Comment by cfobel** [comment]
Trac update at 20120107T01:22:44: 
* cfobel commented:

> For now, this can simply point to:
> http://microfluidics.utoronto.ca/microdrop/wiki/UserGuide


* cfobel changed status from "new" to "accepted"

**Comment by cfobel** [comment]
Trac update at 20120107T01:52:30: 
* christian@fobel.net commented:

> commit 97cdd8590b51c58773d2b67a781f87ae8598996e
> Author: Christian Fobel <christian@fobel.net>
> Date:   Fri Jan 6 20:45:06 2012 -0500
> 
>     Add 'Online help' entry in 'Help' menu. Fixes #38


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "accepted" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
